### PR TITLE
two column layout - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-07-02 at 6 33 58 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Yuliia, the spacing of the content in your output does not match the example, please revise and submit a new screenshot with your revision.

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-05 at 11 22 17 PM

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row-header">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:14]: `row-header`? Why not just `row`? You should make a `column` element inside this `row` to control the size of the content, too

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  display: flex;
+}
+
+.column-wrap {
+  flex-wrap: wrap;
+}
+
+.flex {
+  margin-left: 0;
+  flex-basis: 50%;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:16]: Why isn't this on a rule for the `half-column` class?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
In addition to my other comments, you are still missing a rule for `column-three-quarters`

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**